doc/api/cli.md

```diff
@@ -129,6 +129,20 @@ added: v2.1.0
 Prints a stack trace whenever synchronous I/O is detected after the first turn
 of the event loop.
 
+### `--enable-tracing`
+<!-- YAML
+added: v7.1.0
+-->
+
+Enables the collection of trace event tracing information.
+
+### `--enabled-categories=list`
+<!-- YAML
+added: v7.1.0
+-->
+
+A comma separated list of categories that should be traced when trace event
```

Comment:
What kind of categories are actually available?
It's explained in the PR including how to add custom categories.
Should add that info to the docs as well or at least link to more info.

Also you gonna include some docs regarding the `config.json` file passed via `--trace-config=`? 
Like what are the keys and what they mean? I see that in the PR but nowhere as part of the docs (maybe I'm missing something).
Also again not sure if it should be `--trace-config=` vs. just `--trace-config`.